salt/ocdskingfisherarchive/archive.sudoers

```diff
@@ -1,2 +1,3 @@
 archive ALL = (ocdskfs) NOPASSWD: /bin/rm -rf /home/ocdskfs/scrapyd/data/*/*
+archive ALL = (ocdskfs) NOPASSWD: /bin/rm -f /home/ocdskfs/scrapyd/logs/*/*
```

Comment:
No `-r`?

Reply:
No, it only needs to delete one file at a time so recursive mode isn't needed